front/src/components/ToolTip.css

```diff
@@ -0,0 +1,24 @@
+.tooltip {
+    position: relative;
+  }
+  
+  .tooltip .tooltiptext {
+    visibility: hidden;
+    background-color: black;
+    color: #fff;
+    text-align: center;
+    border-radius: 6px;
+    padding: 5px 0px;
+    left:0;
+    top:20px;
+    opacity: 0.8;
+  
+    /* Position the tooltip */
+    position: absolute;
+    z-index: 1;
+  }
+  
+  .tooltip:hover .tooltiptext {
+    visibility: visible;
+  }
+
```

Comment:
To stay consistent with the rest of the app, please use the colors from [colors.js](https://github.com/olav35/liracer-rewrite/blob/master/front/src/colors.js).

```js
colors.layer2Background // This should be the border color of the tooltip
colors.layer1Background // This should be the background color of the tooltip
colors.layer1Foreground // This should the foreground color(text color) of the tooltip
```

Reply:
Okay I will apply that